credsweeper/__main__.py

```diff
@@ -198,6 +203,13 @@ def scan(args: Namespace, content_provider: FilesProvider, json_filename: Option
 
     """
     try:
+        if args.blacklist_path is not None:
+            with open(args.blacklist_path) as f:
+                blacklist_text = f.read()
+            blacklist = [line for line in blacklist_text.split("\n") if line]
```

Comment:
Need test UTF-16 and windows line ending.

Reply:
Modified with 0a7f3d85f4e5df14eac1dab809fed578b6df4d3d